doc/getting-started/read-and-write-data.rst

```diff
@@ -0,0 +1,58 @@
+Reading and writing data
+========================
+
+As soon as your :doc:`RemoteStorage </js-api/remotestorage>` instance is ready
+for action (signaled by the ``ready`` event), we can start reading and writing
+data.
+
+Anonymous mode
+--------------
+
+One of the unique features of rs.js is that users are not required to have
+their storage connected, before using your app [#f1]_. Any data which is
+written to the local store before connecting an account, is automatically
+synced to the remote storage, whenever one is being connected [#f2]_.
+
+Using BaseClient
+----------------
+
+A ``BaseClient`` instance is the main endpoint you will use for interacting
+with a connected storage: listing, reading, creating, updating and deleting
+documents, as well as handling incoming changes.
+
+Check out the :doc:`BaseClient API docs </js-api/base-client>` in order to
+learn about all functions available for reading and writing data and how to use
+them.
+
+There are two options for acquiring a BaseClient instance:
+
+Quick and dirty: creating a client via ``scope()``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This should mainly be used for manually exploring client functions and in
+development. Using :func:`scope`, you can create a new BaseClient scoped to a
+given path::
+
+   const client = remoteStorage.scope('/foo/');
+
+   // List all items in the "foo/" category/folder
+   client.getListing('')
+     .then(listing => console.log(listing));
+
+   // Write some text to "foo/bar.txt"
```

Comment:
The example writes to `quotes.txt`, not `bar.txt`.

Reply:
Good catch, I was changing names back and forth at some point and missed a line.